app/controllers/concerns/hydra/collections_controller_behavior.rb

```diff
@@ -175,7 +175,9 @@ def move_members_between_collections
     # include filters into the query to only include the collection memebers
     def include_collection_ids(solr_parameters, user_parameters)
       solr_parameters[:fq] ||= []
-      solr_parameters[:fq] << Solrizer.solr_name(:collection, :facetable)+':"'+@collection.id+'"'
+
+      # http://search-lucene.com/javadoc/lucene/core/org/apache/lucene/search/DocValuesTermsFilter.html
+      solr_parameters[:fq] << "{!terms f=id method=docValuesTermsFilter}#{@collection.member_ids.join(',')}"
     end
```

Comment:
See this commit: https://github.com/projecthydra-labs/hydra-collections/commit/dd5fcca98555d6460164475b8cb94a8bc069851c